### DISCUSSION TITLE
[Meeting minutes] [2025-01-22] 회의록

### DISCUSSION THREAD

**Discussion by JaeHyun11** [boostcampaitech7/level4-recsys-finalproject-hackathon-recsys-05-lv3#19] [Meeting]
### 주제

문제 정의 및 실험 방향 설정

### 진행 상황

공지원: LightGCN 논문 리뷰를 읽었고, Cold-Start Recommendation towards the Era of Large Language Models (LLMs): A Comprehensive Survey and Roadmap 이것은 최신거라 논문리뷰가 없어서 초록부분만 읽어둔 상태입니다. MultiVAE를 볼 때부터 메타데이터를 활용하는 방법은 임베딩을 할 때 활용하는 방법 밖에 없다고 생각했었는데 다른 방법이 더 있을지 이 논문을 통해서 알아보려고 합니다. 추가로 오늘 멘토링을 들으면서 콜드 아이템, 콜드 유저에 대해 모델을 따로 해도 괜찮을 것 같다는 생각을 했습니다.
김주은: lightGCN 코드 튜닝 중입니다. 하이브리드 모델에 대해 찾아보고 있는데 knowledge-Attention Network for Recommendation에 대해 읽고 있습니다.
모델만 바꿔쓰는게 콜드스타트 문제 해결방안으로 쓸만 할까요?
다른 팀의 질문 내용 요약, 설명.
류지훈: 다른팀에서 4점 이상의 레이블링을 했습니다. 메트릭 고민 필요. 콜드스타트와 기존 성능의 trade-off 분석이 필요합니다.
지식그래프와 lightGCN 그래프의 역할이 다르다고 생각합니다. 지식그래프를 사용하려면 지식그래프도 학습시켜야 됩니다.
박세연: 주방이 터졌습니다. lightGCN 모델을 공부하고 코드를 pytorch로 찾아보려고 하고 있었습니다.
박재현: lightGCN 논문을 찾아보고 임베딩 방법에 대해 찾아보았습니다. 기존 논문 2개를 봤는데 유저-아이템 상호작용만 사용했습니다. 하지만 메타 데이터를 쓴다면 concat해서 사용할 수 있을 것 같습니다.
백상민: lightGCN 논문을 보고 작동을 찾아보고 있었습니다. 메타데이터를 concat해서 사용할 수 있을 것 같은데 lightGCN이 오래 걸렸던 것 같아 걱정됩니다.
완료된 작업:
진행 중인 작업:

### Agenda

- LightGCN 모델의 기본 코드를 가져와서 통일해서 실험하고 loss function이나 metric을 바꿔가면서 실험 진행하기
- pytorch or tensorflow? pytorch로 결정
- 샘플링, split (cold item 설정방법), loss function, metric 정하기
- 일반적인 모델 성능 메트릭 설정 & cold start 메트릭 설정
NDCG@K
Hit@K
Recall@K
Precision@K
MAP@K

- 코드 팀 - 코드에서 활용한 기법들 정리해오기 : 주은, 상민
- 논문 팀 - 활용 가능한 다른 기법, 지표들 조사해오기 : 지원, 지훈, 재현, 세연

### 직면한 문제점 및 해결 방향

문제점:
해결 방법:

### 추가로 다루어 보아야 할 사항
- 다음 주 일정:
주은: 수요일 안됨
세연: 추석연휴 기간동안 양양에

월화목금 중 하루 오프라인 미팅
